R/dvhydrograph-render.R

```diff
@@ -287,6 +300,11 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", minMaxE
                            lty=ifelse(plotItem[['newSet']] == "est", 1, 6), col=ifelse(plotItem[['newSet']] == "est", "orange", "red3")),
                       styles$est_lines)
     ),
+    estimated4Edges = list(
+      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']], 
+                           lty=ifelse(plotItem[['newSet']] == "est", 1, 7), col=ifelse(plotItem[['newSet']] == "est", "black", "red3")),
```

Comment:
isn't the color defined in dvhydrograph-styles.R? I don't think we need to specify it in both places, right?